lib/naxum/src/serve.rs

```diff
@@ -236,7 +235,6 @@ where
                     }
                 }
             }
-            metric!(counter.naxum.pending_message_count = 0);
```

Comment:
Redundant given that we subtract what's in flight. The `failed` and `none` metrics behave the same.